### PR TITLE
improve development guide

### PR DESCRIPTION
**Pull request by rhythm-sharma**
---

**What's this PR do?**
This PR improve development guide

**Any background context you want to provide?**
This PR solves the  issue improve development guide [#341](https://github.com/zulip/zulip-electron/issues/341)

### CODE REVIEW

File: development.md [hunk 1]
```diff
@@ -1,85 +1,162 @@
...
+You can skip this step if you already have Git, GitHub.
+
+Follow our [Git Guide](https://zulip.readthedocs.io/en/latest/git/setup.html) in order to install Git, set up a GitHub account
+
+
+# Install Prerequisites
+
+Jump to:
+
+- [MacOS](https://paper.dropbox.com/doc/Improve-development-guide--AaFMtIWoCHF1TcoqW7Xz_ARdAg-IOXUYkXRC2dLnSmvuk13t#:h2=MacOS)
```
Comment (akashnimare) [inline development.md]: You need to update paper link to GitHub.

**Review by akashnimare — CHANGES_REQUESTED** [review]

File: development.md (hunk 1 above)
Comment (rhythm-sharma) [reply development.md]: oh didn't noticed :sweat_smile:. fixing right now

Comment (rhythm-sharma) [reply development.md]: updated the links, you can review

**Comment by rhythm-sharma** [comment]
@akashnimare  solve the link problem, you can review

**Comment by akashnimare** [comment]
Thanks, @rrhythmsharma.

**Comment by abhigyank** [comment]
* How did we arrive on the `**2GB of available RAM` data? 
* `**Followings are optional yet recommended prerequisites -**` - What is the use of the following installations?

The english looks very informal at some parts and there is a severe issue of incorrect capitalisation in sentences and headings.
@rishig  should probably go through the text once.

**Comment by rhythm-sharma** [comment]
@abhigyank , before running development environment we need to install many prerequisites, so i thought to add `2 GB` line and i took help from  [here](https://github.com/zulip/zulip-electron/wiki/Windows) for windows installation instruction, please tell me the mistakes [here](https://paper.dropbox.com/doc/Improve-development-guide--AaFMtIWoCHF1TcoqW7Xz_ARdAg-IOXUYkXRC2dLnSmvuk13t)

**Comment by vsvipul** [comment]
@akashnimare @rrhythmsharma additionally speaking, why is there a heading "Improve development guide" in the new guide ? Also see the heading "build and run", at least it should be changed to Title Case. The changes could be better formatted i guess.

**Comment by rhythm-sharma** [comment]
It would better if @abhigyan [review](https://paper.dropbox.com/doc/Improve-development-guide--AaFMtIWoCHF1TcoqW7Xz_ARdAg-IOXUYkXRC2dLnSmvuk13t). I will change it then

**Comment by abhigyank** [comment]
@akashnimare  can we revert the commit PR? There are a **lot** of issues with it. It really looks bad as our development guide. 
@rrhythmsharma I went through the dropbox paper and started to review but there are so many things wrong with it in terms of language and grammar, I think it'd be better to start with a new draft.

**Comment by rhythm-sharma** [comment]
I will send another PR, no issues

**Comment by akashnimare** [comment]
> can we revert the commit PR? There are a lot of issues with it. It really looks bad as our development guide.

I knew this while merging the PR, it was getting difficult to collaborate on paper so I thought let's merge the v1 and then we can take things from there. @vsvipul, @rrhythmsharma feel free to send a PR.

**Pull request merged**